test/parallel_api/ranges/std_ranges_destroy.pass.cpp

```diff
@@ -0,0 +1,50 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/memory>
+#include <oneapi/dpl/ranges>
+
+#include "support/test_config.h"
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if _ENABLE_STD_RANGES_TESTING
+#    include "std_ranges_memory_test.h"
+#endif //_ENABLE_STD_RANGES_TESTING
+
+std::int32_t
```

Comment:
```suggestion
int
```

Somewhere we have `int main`, in other places we have `int32_t`. I prefer us to move towards `int main` because that's how it should be according to the standard and how LLVM folks modified our tests (while PSTL upstream was alive)